.github/workflows/release.yml

```
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+env:
+  node_version: 20
+  pnpm_version: 8
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v3
+        with:
+          version: ${{ env.pnpm_version }}
+      - name: Use Node.js
+        uses: actions/setup-node@v3
```

updating action versions to their latests
```suggestion
      - uses: actions/checkout@v4
      - uses: pnpm/action-setup@v3
        with:
          version: ${{ env.pnpm_version }}
      - name: Use Node.js
        uses: actions/setup-node@v4
```